package.json

```diff
@@ -9,6 +9,7 @@
   "dependencies": {
     "chalk": "^5.3.0",
     "cli-spinner": "^0.2.10",
-    "shelljs": "^0.8.5"
+    "shelljs": "^0.8.5",
+    "tailwindcss": "^3.3.3"
```

Comment:
Why did you add tailwindcss in root? it is only needed in frontend.

Reply:
my bad, i was in wrong directory when i installed it. ill push again with these lines deleted.